mitosheet/mitosheet/user/location.py

```diff
@@ -79,6 +79,8 @@ def get_location() -> str:
     elif is_jupyterlite():
         return 'location_jupyterlite'
     elif notebook and (lab_running and not notebook_running):
+        # NOTE: Since Notebook 7 runs on jlab, we end up in this case
```

Comment:
Update this to just be `jupyter` or figure out how to differentiate which UI that user is looking at